sklearn/feature_selection/_rfe.py

```diff
@@ -57,9 +58,13 @@ class RFE(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
         information about feature importance
         (e.g. `coef_`, `feature_importances_`).
 
-    n_features_to_select : int or None, default=None
-        The number of features to select. If `None`, half of the features
-        are selected.
+    n_features_to_select : int, float, or None (default=None)
+        The number of features to select.
+        If `None`, half of the features are selected.
+        If integer, the parameter is the absolute number of features to select.
+        If float between 0 and 1, it is the fraction of features to select.
+        If float larger than 1, it is converted to integer and represents the
+        absolute number of features to select.
```

Comment:
I think we should error in these cases. This is consistent with the behavior we have in trees and `min_samples_split`.